application/config.json.template

```diff
@@ -44,5 +44,9 @@
            "Other"
        ],
        "categoryRoleSuffix": " - Helper"
-   }
+   },
+  "blacklistedFileExtension": [
+    "exe",
+    "tar"
```

Comment:
Nothing wrong with tar, isnt it? Just an archive, like a zip, not auto-runnable.
Dangerous are things like msi files. Stuff that actually causes harm if executed

Reply:
just an example can change it 😄